src/sagemaker_server.h

```diff
@@ -60,7 +60,8 @@ class SagemakerAPIServer : public HTTPAPIServer {
       triton::server::TraceManager* trace_manager,
       const std::shared_ptr<SharedMemoryManager>& shm_manager,
       const int32_t port, const int thread_cnt)
-      : HTTPAPIServer(server, trace_manager, shm_manager, port, thread_cnt),
+      : HTTPAPIServer(
```

Comment:
Do not set default magic values here. These should be done in main.cc and commented to explain why